back/.gitignore

```diff
@@ -33,4 +33,6 @@ lerna-debug.log*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json
```

Comment:
It's only now that I see those files.
I think we don't want these exceptions to be in here :upside_down_face:

Reply:
woops :eyes: